CHANGELOG.yml

```
@@ -4,6 +4,16 @@ upcoming:
   dev:
     -
   user_facing:
+    - Add artist past shows screen - mounir
+    - Add loading spinner to infinite artwork grid on artist page - david
+    - Avoid shrinking the background on non-fullscreen fancy modals - david
+    - Support iOS 14 photo picker in My Collection - brian
+    - Scroll to add message input on Inquire screen - ashley, mounir
+    - Make add message input height relative to its size - ashley, mounir
+    - Avoid shrinking the background on non-fullscreen fancy modals - david
+    - removed x from inquiry sent notification - lily
+    - Minor copy changes for My Collection - pepopowitz
+    - Move shows entry point to about tab - mounir
```

```suggestion
    - Move shows entry point to about tab - mounir
```
I think only this is needed?